packages/fxa-content-server/app/scripts/views/post_verify/password/force_password_change.js

```diff
@@ -0,0 +1,105 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * This view handles forcing a user to change their password.
+ */
+import Cocktail from 'cocktail';
+import FormView from '../../form';
+import FlowEventsMixin from './../../mixins/flow-events-mixin';
+import PasswordMixin from '../../mixins/password-mixin';
+import PasswordStrengthMixin from '../../mixins/password-strength-mixin';
+import ServiceMixin from '../../mixins/service-mixin';
+import Template from 'templates/post_verify/password/force_password_change.mustache';
+import VerificationReasonMixin from '../../mixins/verification-reason-mixin';
+import AuthErrors from '../../../lib/auth-errors';
+
+const OPASSWORD_INPUT_SELECTOR = '#opassword';
+const PASSWORD_INPUT_SELECTOR = '#password';
+const VPASSWORD_INPUT_SELECTOR = '#vpassword';
+
+class ForcePasswordChange extends FormView {
+  template = Template;
+  viewName = 'force-password-change';
+
+  _getOldPassword() {
+    return this.$(OPASSWORD_INPUT_SELECTOR).val();
+  }
+
+  _getPassword() {
+    return this.$(PASSWORD_INPUT_SELECTOR).val();
+  }
+
+  _getVPassword() {
+    return this.$(VPASSWORD_INPUT_SELECTOR).val();
+  }
+
+  setInitialContext(context) {
+    console.log('setInitialContext', context);
+    const account = this.getSignedInAccount();
+    const email = account.get('email');
+    context.set({
+      email,
+    });
+  }
+
+  getAccount() {
+    return this.getSignedInAccount();
+  }
```

Comment:
Is this function required by something else?  I see `this.getSignedInAccount` used directly elsewhere in this view.

Reply:
Oh, mixins.